quickstart/mutators.markdown

```diff
@@ -758,9 +765,134 @@ The switch mutator finds the first label within a switch statement that differs
 
 *Thanks to Stefan Penndorf who contributed this documentation.*
```

Comment:
I think that this "thanks to" line was meant to go at the bottom of the documentation (right before the final `<hr/>`) and that it should stay there.

(I don't actually know why he was credited this way, but I guess that he contributed the original version of the documentation or a substantial part of it. At some point, after enough changes from enough people, I think that the git history is a better way to known who contributed to what, but this is not a topic for _this_ PR :smile: )